src/xo-app/home/pool-item.js

```diff
@@ -44,8 +44,17 @@ import styles from './index.css'
     }
   )
 
+  const getHostMetrics = createSelector(
+    getPoolHosts,
+    hosts => ({
+      count: size(hosts),
+      cpus: reduce(hosts, (total, host) => total + +host.cpus.cores, 0),
+      memory: reduce(hosts, (total, host) => total + +host.memory.size, 0)
+    })
```

Comment:
Isn't it possible to mutualize code with [Dashboard/Overview](https://github.com/vatesfr/xo-web/blob/next-release/src/xo-app/dashboard/overview/index.js#L190-L207)?